.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: node_js
+node_js:
+  - 'node'
+  - '6'
+  - '4'
+addons:
+  code_climate:
+    repo_token: 'your repo token'
+notifications:
+  slack: feathersjs:T4tvpJFt27wJZABjLSg5OqMv
```

Comment:
@corymsmith I'm not sure if it came from an old generator version or was copy pasted. Like the former. So everyone should make sure they are on the latest `generator-feathers-plugin`.